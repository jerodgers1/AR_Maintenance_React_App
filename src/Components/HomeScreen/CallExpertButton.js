import React from 'react';
import { Modal, View, StyleSheet, Text,} from 'react-native';
import { Button } from 'native-base';
import {connect} from "react-redux";

class CallExpertButton extends React.Component {

    state = {
        modalVisible: false,
    };

    openModal() {
        this.setState({modalVisible:true});
    }

    closeModal() {
        this.setState({modalVisible:false});
    }

    callExpertHandler = () => {
        console.log('calling Expert for job ' + this.props.selectedJob);
    }

    render() {
        return (
            <View>
                <Modal
                    visible={this.state.modalVisible}
                    animationType={'slide'}
                    onRequestClose={() => this.closeModal()}
                >
                    <View style={styles.modalContainer}>
                        <View style={styles.innerContainer}>
                            <Text style={{fontSize: 35}}>Call Expert?</Text>
                            <View style={styles.buttonLayout}>
                                <Button onPress={() => this.closeModal()} style={styles.cancelButton} large rounded>
                                    <Text style={{marginLeft: '15%', fontSize: 20}}> Cancel </Text>
                                </Button>
                                <Button onPress={() => this.callExpertHandler()} style={styles.callButton} large rounded>
                                    <Text style={{marginLeft: '25%', fontSize: 20}}> Call </Text>
                                </Button>
                            </View>
                        </View>
                    </View>
                </Modal>
                <Button onPress={() => this.openModal()} style={styles.expertButton} large rounded>
                    <Text style={{marginLeft: '10%', fontSize: 20}}> Call Expert </Text>
                </Button>
            </View>
        );
    }
}

const styles = StyleSheet.create({
    expertButton: {
        marginLeft: 2,
        width: 140,
    },
    callButton:{
        width: 100,
        margin:'5%',
    },
    cancelButton:{
        backgroundColor: '#f93831',
        width: 100,
        margin:'5%',
    },
    buttonLayout: {
        flex: 1,
        flexDirection: 'row',
    },
    modalContainer: {
        flex: 1,
        justifyContent: 'center',
        backgroundColor: 'grey',
    },
    innerContainer: {
        alignItems: 'center',
        marginTop: '-20%',
    },

});

const mapStateToProps = state => {
    return {
        selectedJob: state.jobs.selectedJob
    };
};

const mapDispatchToProps = dispatch => {
    return {
    };
};

export default connect(mapStateToProps, mapDispatchToProps)(CallExpertButton);
