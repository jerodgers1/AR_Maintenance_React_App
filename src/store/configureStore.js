import { createStore, combineReducers } from 'redux';

import jobsReducer from './reducers/jobsReducers';

const rootReducer = combineReducers({
    jobs: jobsReducer
});

const configureStore = () => {
    return createStore(rootReducer);
};

export default configureStore;