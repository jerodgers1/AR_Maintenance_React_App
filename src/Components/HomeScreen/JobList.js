import React from 'react';
import { StyleSheet, Text, View, ListView} from 'react-native';
import { connect } from "react-redux";
import {selectJob} from "../../store/actions";

class JobList extends React.Component {

    constructor(props) {
        super(props);
        const ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2});
        this.state = {
            isReady: false,
            dataSource: ds.cloneWithRows(this.props.jobs),
        };
    }

    selectJobHandler = selected => {
        this.props.onSelectJob(selected);
        console.log(this.props.selectedJob);
    };

    i = 1;

    render() {
        return (
            <View style={styles.container}>
                <ListView
                    style={styles.list}
                    dataSource={this.state.dataSource}
                    renderRow={(rowData) => <Text onPress={() => this.selectJobHandler(rowData)} style={styles.listItems}>
                        {this.i++}) { rowData }
                    </Text>
                    }
                />
            </View>
        );
    }
}

const styles = StyleSheet.create({
    list: {
        paddingLeft: 40,
    },
    listItems: {
        fontSize: 25,
        paddingTop: 10,
        color: 'black',
        textShadowOffset: {width: 1, height: 1}
    },
    container: {
        width: '35%',
        height: '100%',
        alignSelf: 'flex-start',
        borderRightWidth: 1,
        borderRightColor: '#7c7f99'

    },
});



const mapStateToProps = state => {
    return {
        jobs: state.jobs.jobs,
        selectedJob: state.jobs.selectedJob
    };
};

const mapDispatchToProps = dispatch => {
    return {
        onSelectJob: job => dispatch(selectJob(job)),
    };
};

export default connect(mapStateToProps, mapDispatchToProps)(JobList);

