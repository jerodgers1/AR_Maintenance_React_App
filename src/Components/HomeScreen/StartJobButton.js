import React from 'react';
import {View, StyleSheet, Text, ListView} from 'react-native';
import { Button } from 'native-base';
import {connect} from "react-redux";
import {selectJob} from "../../store/actions";
import {Navigation} from "react-native-navigation";

const StartJobButton = props => {
        return (
            <View>
                <Button onPress={() => props.onJobStarted()} style={styles.startButton} large rounded>
                    <Text style={{marginLeft: '25%', fontSize: 20}}> Start </Text>
                </Button>
            </View>
        );
}

const styles = StyleSheet.create({
    startButton: {
        marginLeft: '30%',
        width: 140,
    },
});
export default StartJobButton;
// const mapStateToProps = state => {
//     return {
//         selectedJob: state.jobs.selectedJob
//     };
// };
//
// const mapDispatchToProps = dispatch => {
//     return {
//        // StartJob: job => dispatch(StartJob(job)),
//     };
// };
//
// export default connect(mapStateToProps, mapDispatchToProps)(StartJobButton);
//
