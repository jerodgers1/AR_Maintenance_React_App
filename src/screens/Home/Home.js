import React from 'react';
import { StyleSheet, Text, View, ListView} from 'react-native';
import { connect } from "react-redux";
import { Container, Header, Title, Button, Left, Right, Body, Icon, Drawer } from 'native-base';
//redux
import { addPlace, deletePlace, selectPlace, deselectPlace} from "../../store/actions/index";

import JobList from '../../Components/HomeScreen/JobList.js';
import CallExpertButton from "../../Components/HomeScreen/CallExpertButton";
import  StartJobButton  from "../../Components/HomeScreen/StartJobButton";
import {addJob} from "../../store/actions";

class Home extends React.Component {

    constructor(props) {
        super(props);
        this.state = {
        };

    }

    startJobScreen = ()=> {
        this.props.navigator.push({
            screen: "AR_Maintenace_App.JobScreen",
            title: this.props.selectedJob,
            navigatorStyle: {navBarBackgroundColor: '#c8c8c8'}
        });
    };

    render() {
        return (
            <Container>
                {/*welcome screen*/}
                <View style={styles.container}>
                    {/*left side of the app*/}
                    <JobList />
                    {/*This is how you would pass props if I wasn't using Redux*/}
                    {/*<JobList jobList={this.state.jobs}/>*/}
                    <View style={styles.mainScreen}>
                        <View style={styles.messageView}>
                            <Text style={styles.message}>Welcome to AR Maintenance</Text>
                        </View>
                        {this.props.showButtons ? (<View style={styles.mainMenuButtons}>
                            <CallExpertButton />
                            <StartJobButton onJobStarted={this.startJobScreen}/>
                        </View>) : (<View style={{height: '100%'}}/>)}
                    </View>
                </View>
            </Container>
        );
    }
}

const styles = StyleSheet.create({
    mainScreen: {
        width: '60%',
        flex: 1,
        flexDirection: 'column',
        backgroundColor: '#c8c8c8',
        justifyContent: 'flex-start',
    },
    mainMenuButtons:{
        flex: 1,
        flexDirection: 'row',
        justifyContent: 'flex-start',
        backgroundColor: '#c8c8c8',
        paddingTop: 30,
        paddingLeft: '22%',
    },
    message: {
        textAlign: 'center',
        fontSize: 44,
        color: '#2b70ec',
        fontFamily: 'Roboto',
        textShadowOffset: {width: 2, height: 2},
    },
    messageView: {
        paddingTop: '15%',
    },
    container: {
        width: '100%',
        flex: 1,
        flexDirection: 'row',
        backgroundColor: '#cecece',
        alignItems: 'flex-start',
    },
});


const mapStateToProps = state => {
    return {
        showButtons: state.jobs.showButtons,
        selectedJob: state.jobs.selectedJob,
        jobList: state.jobs.jobs
    };
};

const mapDispatchToProps = dispatch => {
    return {
    };
};

export default connect(mapStateToProps, mapDispatchToProps)(Home);
