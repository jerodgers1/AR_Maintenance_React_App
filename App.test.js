import React from 'react';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';

import Home from './src/screens/Home/Home';
import CallExpertButton from './src/Components/HomeScreen/CallExpertButton';
import StartJobButton from './src/Components/HomeScreen/StartJobButton';
import JobList from './src/Components/HomeScreen/JobList';

import configureStore from './src/store/configureStore';
const store = configureStore();

test('Home', () => {
  const component = renderer.create(
      <Provider store={store}>
            <Home />
      </Provider>
  ).toJSON();
  expect(component).toMatchSnapshot();
});

test('Expert Button', () => {
    const component = renderer.create(
        <Provider store={store}>
            <CallExpertButton/>
        </Provider>
    ).toJSON();
    expect(component).toMatchSnapshot();
});

test('start Button', () => {
    const component = renderer.create(
        <StartJobButton onJobStarted={console.log('test')}/>
    ).toJSON();
    expect(component).toMatchSnapshot();
});

test('Job List', () => {
    const component = renderer.create(
        <Provider store={store}>
            <JobList jobList={"test"}/>
        </Provider>
    ).toJSON();
    expect(component).toMatchSnapshot();
});
