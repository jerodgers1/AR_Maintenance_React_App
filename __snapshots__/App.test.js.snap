// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expert Button 1`] = `
<View>
  <Modal
    animationType="slide"
    hardwareAccelerated={false}
    onRequestClose={[Function]}
    visible={false}
  >
    <View
      style={
        Object {
          "backgroundColor": "grey",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "marginTop": "-20%",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "fontSize": 35,
            }
          }
        >
          Call Expert?
        </Text>
        <View
          style={
            Object {
              "flex": 1,
              "flexDirection": "row",
            }
          }
        >
          <View
            accessibilityComponentType={undefined}
            accessibilityLabel={undefined}
            accessibilityTraits={undefined}
            accessible={true}
            collapsable={undefined}
            hasTVPreferredFocus={undefined}
            hitSlop={undefined}
            isTVSelectable={true}
            nativeID={undefined}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "flex-start",
                "backgroundColor": "#f93831",
                "borderBottomWidth": null,
                "borderColor": "#007aff",
                "borderLeftWidth": null,
                "borderRadius": 57,
                "borderRightWidth": null,
                "borderTopWidth": null,
                "elevation": 2,
                "flexDirection": "row",
                "height": 60,
                "justifyContent": "space-between",
                "margin": "5%",
                "opacity": 1,
                "paddingBottom": 6,
                "paddingTop": 6,
                "width": 100,
              }
            }
            testID={undefined}
            tvParallaxProperties={undefined}
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontSize": 20,
                  "marginLeft": "15%",
                }
              }
            >
               Cancel 
            </Text>
          </View>
          <View
            accessibilityComponentType={undefined}
            accessibilityLabel={undefined}
            accessibilityTraits={undefined}
            accessible={true}
            collapsable={undefined}
            hasTVPreferredFocus={undefined}
            hitSlop={undefined}
            isTVSelectable={true}
            nativeID={undefined}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "alignSelf": "flex-start",
                "backgroundColor": "#007aff",
                "borderBottomWidth": null,
                "borderColor": "#007aff",
                "borderLeftWidth": null,
                "borderRadius": 57,
                "borderRightWidth": null,
                "borderTopWidth": null,
                "elevation": 2,
                "flexDirection": "row",
                "height": 60,
                "justifyContent": "space-between",
                "margin": "5%",
                "opacity": 1,
                "paddingBottom": 6,
                "paddingTop": 6,
                "width": 100,
              }
            }
            testID={undefined}
            tvParallaxProperties={undefined}
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Object {
                  "fontSize": 20,
                  "marginLeft": "25%",
                }
              }
            >
               Call 
            </Text>
          </View>
        </View>
      </View>
    </View>
  </Modal>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "flex-start",
        "backgroundColor": "#007aff",
        "borderBottomWidth": null,
        "borderColor": "#007aff",
        "borderLeftWidth": null,
        "borderRadius": 57,
        "borderRightWidth": null,
        "borderTopWidth": null,
        "elevation": 2,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "space-between",
        "marginLeft": 2,
        "opacity": 1,
        "paddingBottom": 6,
        "paddingTop": 6,
        "width": 140,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 20,
          "marginLeft": "10%",
        }
      }
    >
       Call Expert 
    </Text>
  </View>
</View>
`;

exports[`Home 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "height": 1334,
    }
  }
  virtual={undefined}
>
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "backgroundColor": "#cecece",
        "flex": 1,
        "flexDirection": "row",
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "alignSelf": "flex-start",
          "borderRightColor": "#7c7f99",
          "borderRightWidth": 1,
          "height": "100%",
          "width": "35%",
        }
      }
    >
      <RCTScrollView
        dataSource={
          ListViewDataSource {
            "items": 14,
          }
        }
        renderRow={[Function]}
        renderScrollComponent={[Function]}
        style={
          Object {
            "paddingLeft": 40,
          }
        }
      >
        <View>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            1
            ) 
            Change Oil
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            2
            ) 
            Change starter
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            3
            ) 
            Rotate Tire
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            4
            ) 
            Check Timing Belt
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            5
            ) 
            Check Fluid levels
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            6
            ) 
            Check lights
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            7
            ) 
            Check Brake lines
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            8
            ) 
            Check Brake pads
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            9
            ) 
            Rotate Tire
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            10
            ) 
            Check Timing Belt
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            11
            ) 
            Check Fluid levels
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            12
            ) 
            Check lights
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            13
            ) 
            Check Brake lines
          </Text>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Object {
                "color": "black",
                "fontSize": 25,
                "paddingTop": 10,
                "textShadowOffset": Object {
                  "height": 1,
                  "width": 1,
                },
              }
            }
          >
            14
            ) 
            Check Brake pads
          </Text>
        </View>
      </RCTScrollView>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#c8c8c8",
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "flex-start",
          "width": "60%",
        }
      }
    >
      <View
        style={
          Object {
            "paddingTop": "15%",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#2b70ec",
              "fontFamily": "Roboto",
              "fontSize": 44,
              "textAlign": "center",
              "textShadowOffset": Object {
                "height": 2,
                "width": 2,
              },
            }
          }
        >
          Welcome to AR Maintenance
        </Text>
      </View>
      <View
        style={
          Object {
            "height": "100%",
          }
        }
      />
    </View>
  </View>
</View>
`;

exports[`Job List 1`] = `
<View
  style={
    Object {
      "alignSelf": "flex-start",
      "borderRightColor": "#7c7f99",
      "borderRightWidth": 1,
      "height": "100%",
      "width": "35%",
    }
  }
>
  <RCTScrollView
    dataSource={
      ListViewDataSource {
        "items": 14,
      }
    }
    renderRow={[Function]}
    renderScrollComponent={[Function]}
    style={
      Object {
        "paddingLeft": 40,
      }
    }
  >
    <View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        1
        ) 
        Change Oil
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        2
        ) 
        Change starter
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        3
        ) 
        Rotate Tire
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        4
        ) 
        Check Timing Belt
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        5
        ) 
        Check Fluid levels
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        6
        ) 
        Check lights
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        7
        ) 
        Check Brake lines
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        8
        ) 
        Check Brake pads
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        9
        ) 
        Rotate Tire
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        10
        ) 
        Check Timing Belt
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        11
        ) 
        Check Fluid levels
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        12
        ) 
        Check lights
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        13
        ) 
        Check Brake lines
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        onPress={[Function]}
        style={
          Object {
            "color": "black",
            "fontSize": 25,
            "paddingTop": 10,
            "textShadowOffset": Object {
              "height": 1,
              "width": 1,
            },
          }
        }
      >
        14
        ) 
        Check Brake pads
      </Text>
    </View>
  </RCTScrollView>
</View>
`;

exports[`start Button 1`] = `
<View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "alignSelf": "flex-start",
        "backgroundColor": "#007aff",
        "borderBottomWidth": null,
        "borderColor": "#007aff",
        "borderLeftWidth": null,
        "borderRadius": 57,
        "borderRightWidth": null,
        "borderTopWidth": null,
        "elevation": 2,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "space-between",
        "marginLeft": "30%",
        "opacity": 1,
        "paddingBottom": 6,
        "paddingTop": 6,
        "width": 140,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 20,
          "marginLeft": "25%",
        }
      }
    >
       Start 
    </Text>
  </View>
</View>
`;
